test/conformance/serial/ptp.go

```diff
@@ -45,11 +46,6 @@ const (
 	masterOffsetLowerBound  = -100
```

Comment:
future changes , it will be good if we can configure this outside too , via env variable may before basic (default) test cases

Reply:
We can put these also under the global part of the test config, no?